### DISCUSSION TITLE
Case-cohort designs

### DISCUSSION THREAD

**Discussion by feichel** [genetics-of-dna-methylation-consortium/godmc_phase2#7] [Q&A] [answered]
Dear developers-team,
How do you prefer us to analyze a case-cohort design with the pipeline? Should we only consider the subcohort (random 5%-subsample of full cohort ~1100 samples) or also include all external incident disease cases (~600 samples) in order to increase sample size? In the latter case, I would assume we follow your recommendation for case-control studies and combine cases and controls? Should we add an indicator for case status at any point of the analysis?
Thanks!

**Comment by epzjlm** [comment, marked as answer]
We would recommend to combine cases and controls. The methylation PCs that are calculated in script 03 and will be added as covariates will capture some diseases, but not all. So I agree that adding an extra indicator for case/control status as covariate might be a good idea.
